src/erpe/pom.xml

```diff
@@ -262,6 +277,29 @@
           </execution>
         </executions>
       </plugin>
+      
+     <plugin>
+           <artifactid>maven-surefire-plugin</artifactid>
+            <version>2.19</version>
+              <dependencies>
+                <dependency>
+                   <groupid>org.junit.platform</groupid>
+                    <artifactid>junit-platform-surefire-provider</artifactid>
+                   <version>1.0.0-M3</version>
+               </dependency>
+               <dependency>
+                    <groupid>org.junit.jupiter</groupid>
+                   <artifactid>junit-jupiter-engine</artifactid>
+                   <version>5.0.0-M3</version>
+                </dependency>
+                <dependency>
+                    <groupid>org.junit.vintage</groupid>
+                    <artifactid>junit-vintage-engine</artifactid>
+                   <version>4.12.0-M3</version>
+                </dependency>
+           </dependencies>
+        </plugin>
+      
```

Comment:
All this are plugins which are necessary to use JUnit5 which isn't  official verison.
After official release there will be required changes to official librar.